packages/react-jss/src/JssProvider.js

```diff
@@ -18,7 +18,7 @@ type Props = {
   children: Node
 }
 
-export default class JssProvider extends Component<Props> {
```

Comment:
same question here ... the naming convention for file names was this:

If a default export is a class use Capital case file name.  Also use a default export if it's the only export.

Reply:
Fixed